### ISSUE TITLE
Comments for jQuery contents() method

### ISSUE THREAD

**Issue by phinjensen** [EndPointCorp/end-point-blog#872]
Comments for https://www.endpointdev.com/blog/2013/11/jquery-contents-method/
By Steph Skardal

To enter a comment:
1. Log in to GitHub
2. Leave a comment on this issue.

**Comment by DanHashemi** [comment]
Could this work for an accessibility tool to enable high contrast so that when it is clicked, ALL elements that have text has wrapped in a span with class "accessible" for example which adds a black background and makes the text white to stand out?

I tried your code but wasn't successful in making this happen.
Any thoughts or things I'm not considering?
